ppl/archive/schema.go

```diff
@@ -18,9 +18,10 @@ const metadataFilename = "zar.json"
 type Metadata struct {
 	Version int `json:"version"`
 
-	DataPath         string     `json:"data_path"`
-	DataOrder        zbuf.Order `json:"data_order"`
-	LogSizeThreshold int64      `json:"log_size_threshold"`
+	DataPath           string        `json:"data_path"`
+	DataOrder          zbuf.Order    `json:"data_order"`
+	ImportFlushTimeout time.Duration `json:"import_flush_time"`
```

Comment:
The import flush timeout doesn't feel like an archive-defining value the same way that the log size threshold does. Instead of storing it in the archive metadata, it could have a default that could be specified at the import call.

Reply:
Agreed.